### PR TITLE
Updated tasks doc

### PR DESCRIPTION
**Pull request by mans2singh**
Signed-off-by: Mans Singh <mans2singh@yahoo.com>

**What this PR does / why we need it**:

Corrected documentation for tasks

### CODE REVIEW

File: content/docs/developing-operators/tasks.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
 
 ## Overview
 
-A task is the basic building block in the KUDO workflow. Plans, phases, and steps are control structures that execute tasks at the end. You've already come across an Apply-task when developing your [first-operator](getting-started.md). KUDO (as of 0.9.0) offers three main task types: `Apply`, `Delete`, `Pipe` and `Toggle`. Additionally there is a `Dummy` task which is helpful when debugging and testing your operator. All KUDO tasks are defined in the [operator.yaml](packages.md) and must have three fields:
+A task is the basic building block in the KUDO workflow. Plans, phases, and steps are control structures that execute tasks at the end. You've already come across an Apply-task when developing your [first-operator](getting-started.md). KUDO (as of 0.9.0) offers following main task types: `Apply`, `Delete`, `Pipe` and `Toggle`. Additionally there is a `Dummy` task which is helpful when debugging and testing your operator. All KUDO tasks are defined in the [operator.yaml](packages.md) and must have three fields:
```
Comment (meichstedt) [inline content/docs/developing-operators/tasks.md]:
```suggestion
A task is the basic building block in the KUDO workflow. Plans, phases, and steps are control structures that execute tasks at the end. You've already come across an Apply-task when developing your [first-operator](getting-started.md). KUDO offers the following main task types: `Apply`, `Delete`, `Pipe` and `Toggle`. Additionally there is a `Dummy` task which is helpful when debugging and testing your operator. All KUDO tasks are defined in the [operator.yaml](packages.md) and must have three fields:
```

If I'm not mistaken `Toggle` tasks were introduced in [0.11.0](https://github.com/kudobuilder/kudo/releases/tag/v0.11.0), so I'd just remove the version here for simplicity.

**Review by meichstedt — COMMENTED** [review]
Thanks @mans2singh! One tiny suggestion, please see above.

File: content/docs/developing-operators/tasks.md (hunk 1 above)
Comment (mans2singh) [reply content/docs/developing-operators/tasks.md]: @meichstedt - 

I've removed the version based on your comments.  Please let me know if there is any other advice/recommendations. 

Thanks for your review.

**Review by meichstedt — APPROVED** [review]
Thanks @mans2singh!

**Pull request merged**